src/routes/volume/CreateVolume.js

```diff
@@ -128,6 +129,15 @@ const modal = ({
           { frontends.map(opt => <Option key={opt.value} value={opt.value}>{opt.label}</Option>) }
           </Select>)}
         </FormItem>
+        <Spin spinning={tagsLoading}>
+          <FormItem label="Data Locality" hasFeedback {...formItemLayout}>
+            {getFieldDecorator('dataLocality', {
+              initialValue: 'best-effort',
```

Comment:
Should this be the same setting from the `setting.DataLocality`?

Reply:
Oops. Yes, it should be `setting.DataLocality `. I am sorry, my bad

Reply:
Change to `settings. defaultDataLocality.value`.  When the user changes `setting.DataLocality.value`, the default value of the created volume will change as well!  Is this okay ? @yasker @PhanLe1010

Reply:
@smallteeths 
I think the logic looks good.
I am wondering why you don't put the `defaultDataLocalityOption ` into the same `item` object as what we did for `defaultNumberOfReplicas` here https://github.com/longhorn/longhorn-ui/blob/c64b23ecccd0da02368ce958e72154c8758f79c2/src/routes/volume/index.js#L600 ? Is there a problem if we do so?

Reply:
I feel like I can be more standardized by not passing it in the item, because I can judge what type it's passing in. Because longhorn has been through a lot of people, some of the writing is not consistent. I'll pay attention to that and change it over time. @PhanLe1010  

![Jietu20200903-093940](https://user-images.githubusercontent.com/27733391/92061668-6e1fab00-edc9-11ea-84fc-8df27b926f65.jpg)

Reply:
Thank you @smallteeths for the explanation :)